games/tic_tac_toe.c

```diff
@@ -42,8 +42,16 @@ int main()
     {
         int n = 0;
 
-        // filling the table with multiple asterisks
-        for (int i = 0; i < 9; i++) game_table[i] = '*';
+        // filling the table with indices
+	game_table[0] = '1';
```

Comment:
I'm not sure what's the difference between this and the previous code. Could you explain, please?